lms/templates/ccx/schedule.html

```diff
@@ -29,80 +29,88 @@
   <div id="new-ccx-schedule"></div>
 </div>
 
-<section id="enter-date-modal" class="modal" aria-hidden="true">
-  <div class="inner-wrapper" role="dialog">
-    <button class="close-modal">
-      <i class="icon fa fa-remove"></i>
+<section id="enter-date-modal" class="modal"
+         tabindex="1" role="dialog" aria-labelledby="ccx_schedule_set_date_heading">
+  <div class="inner-wrapper">
+    <button class="close-modal" tabindex="2">
```

Comment:
again, you should never use a positive integer.  buttons are already focusable, so no ```tabindex``` is required.